tuftool/CHANGELOG.md

```diff
@@ -4,6 +4,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.0] - 2019-12-16
+### Added
+- Added integration test `create_verify_repo`, which creates a TUF repo with `tuftool` and verifies we can read its targets with `tough`.
+
+### Changed
+- Updated tough to v0.3
+- Changed `RootKeys` to be a `HashMap<Decoded<Hex>, Box<dyn Sign>>` to remove a reference to `KeyPair` (see "Removed" section)
+- `KeySource` now contains an `as_sign` method that returns `Result<Box<dyn Sign>>`.
+
+### Removed
+- Remove `KeyPair` enum from `tuftool` and updated any areas that reference this enum. This is a breaking change.
```

Comment:
This is not a breaking change because none of tuftool's innards are public. (But we are bumping to v0.2 since a lot is changing.)